rules/no-hex-escape.js

```diff
@@ -10,7 +10,9 @@ function checkEscape(context, node, value) {
 		context.report({
 			node,
 			message: 'Use Unicode escapes instead of hexadecimal escapes.',
-			fix: fixer => fixer.replaceTextRange([node.start, node.end], fixedValue)
+			fix: fixer => typeof node.start === 'undefined' ?
+				fixer.replaceText(node, fixedValue) :
+				fixer.replaceTextRange([node.start, node.end], fixedValue)
```

Comment:
Can you drop the conditional? We can just switch completely to `fixer.replaceText`.

Reply:
I don't know why but it throws if I switch completely.

Reply:
```js
 Node {
    type: 'TemplateElement',
    start: 13,
    end: 16,
    range: [ 12, 18 ]
  },
```

seem `TemplateElement` has different start end